requirements.txt

```diff
@@ -2,4 +2,4 @@ grpcio>=1.53.0
 protobuf>=4.22.1
 PyJWT>=2.6.0
 python-ulid>=1.1.0
-requests>=2.28.2
\ No newline at end of file
+requests>=2.25.1,<2.30.0
```

Comment:
See https://github.com/urllib3/urllib3/issues/2168, requests 2.30.0 allows urllib3 v2 but that version of urllib3 discontinued support for OpenSSL < 1.1.1 which has compatibility issues with Python < 3.10